packages/theme/src/colors/token.ts

```diff
@@ -18,6 +18,14 @@ import { grey10, grey20, grey30, grey40, grey60, grey80 } from './secondary'
  */
 export const brand = blue60()
 
+/**
+ * Shade
+ *
+ * {@link https://zeroheight.com/3ddd0f892/p/028ae9-colors/b/036155/t/492c47}
+ */
+export const shadeBlack = '#000000'
+export const shadeWhite = '#FFFFFF'
```

Comment:
Worth reusing this constant in `surfacePrimary`?

Reply:
Updated!